core/src/com/unciv/logic/automation/SpecificUnitAutomation.kt

```diff
@@ -280,6 +287,76 @@ object SpecificUnitAutomation {
         }
     }
 
+    fun automateMissionary(unit: MapUnit){
+        val cities = unit.civInfo.gameInfo.getCities().asSequence()
+            .filter { it.religion.getMajorityReligion()?.name != unit.getReligionDisplayName() }
```

Comment:
Do note that the religion of the missionary may not be the same as the religion of the civ owning it. If you capture a missionary or great prophet, it retains its original religion

Reply:
Additionally, you seem to only target your own cities with missionaries, should you not also want to convert cities of other civs?

Reply:
yeah good catch

Reply:
for point 2, unit.civinfo.gameinfo.getcities() represents all the cities in the game, not just the civs cities

Reply:
fixed